patternfly/patternfly-org
# Adds Topology as top-level nav item

Closes #3349 , dependent on https://github.com/patternfly/patternfly-react/pull/8508

### How to test:

In order to see the topology demos under the "Topology" section, you need to make [this change](https://github.com/patternfly/patternfly-react/pull/8586/files) in org under `node_modules/@patternfly/react-topology/src/components/TopologyView/examples/Topology.md`  and then rebuild.
